functions-and-scope.js

```diff
@@ -16,20 +16,30 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6
 
-
 /*  1b: Omschrijven tot een herbruikbare functie   */
 // Schrijf een functie genaamd cumLaude, die een array van cijfers verwacht (zoals grades) en het aantal Cum laude studenten teruggeeft. Gebruik hiervoor jouw antwoord van 1a.
 // Zorg ervoor dat jouw functie ook werkt als we een andere array met eindcijfers willen checken, zoals bijvoorbeeld: [6, 4, 5] of [8, 9, 4, 6, 10].
 // Log het antwoord in de terminal.
 
+function cumLaudeCounter(grades) {
+    const cumLaude = []
+    for (let i = 0; i < grades.length; i++) {
+        if (grades[i] >= 8) {
+            cumLaude.push(grades[i]);
```

Comment:
Keurige oplossing. Je had ook een variabele kunnen maken met een initiële waarde van 0. Vervolgens telkens 1 erbij optellen zodra een cijfer een 8 of hoger is. 

Net iets minder code.